### PR TITLE
bug fix

### PR DESCRIPTION
**Pull request by ludbek**

### CODE REVIEW

**Review by mastfissh — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ function render(template, fillers, values) {
   }
 
   const fills = Object.keys(fillers).reduce((acc, key) => {
-    acc[key] = fillers[key]($, values) || ""
+    if(template.indexOf(`{${key}}`) !== -1) {
```
Comment (ghost) [inline index.js:10]: do we need interpolation?

**Review by ghost — APPROVED** [review]

**Review by forkfork — APPROVED** [review]
:+1:

File: index.js (hunk 1 above)
Comment (ludbek) [reply index.js:10]: yeah we need to convert the variable to a string.

Comment (ludbek) [reply index.js:10]: also surround it with `{}`

**Pull request merged**